packages/ledger-icrc/src/types/index-ng.params.ts

```diff
@@ -7,3 +9,8 @@ export type GetIndexNgAccountTransactionsParams = {
   start?: IcrcNgTxId;
   account: IcrcAccount;
 } & QueryParams;
+
+export type ListSubaccountsParams = {
```

Comment:
What do you think about reusing `IcrcAccount` in the params? Assuming the owner is the owner of the account.

```
export type ListSubaccountsParams = {
   start?: Subaccount;
} Pick<IcrcAccount, "owner"> & & QueryParams
```

Reply:
I believe it feels a bit forced since we are selecting just one property from the type. However, if you think it makes sense, I will change it.

Reply:
After your explanation, it felt more meaningful to use the `IcrcAccount` :)